jupyterlab_git/git.py

```diff
@@ -1750,3 +1750,7 @@ def ensure_git_credential_cache_daemon(
 
         elif self._GIT_CREDENTIAL_CACHE_DAEMON_PROCESS.poll():
             self.ensure_git_credential_cache_daemon(socket, debug, True, cwd, env)
+
+    @property
+    def excluded_paths(self):
+        return self._config.excluded_paths
```

Comment:
Let's add some documentation:

```suggestion
    def excluded_paths(self) -> List[str]:
        """Wildcard-style path patterns that do not support git commands.
        
        You can use ``*`` to match everything or ``?`` to match any single character.
        """
        return self._config.excluded_paths
```